client/landing/stepper/declarative-flow/link-in-bio.ts

```diff
@@ -15,7 +15,7 @@ export const linkInBio: Flow = {
 			recordTracksEvent( 'calypso_signup_start', { flow: this.name } );
 		}, [] );
 
-		return [ 'intro', 'linkInBioSetup', 'patterns', 'processing', 'launchpad' ] as StepPath[];
+		return [ 'intro', 'linkInBioSetup', 'patterns', 'launchpad' ] as StepPath[];
```

Comment:
I think we need to leave the processing here, or remove this submit, since it uses this
```				case 'launchpad': {
					return navigate( 'processing' );
				}
```

Reply:
100%